### ISSUE TITLE
Add Quaaykop

### ISSUE THREAD

**Issue by ComteDuChagrin** [google/fonts#6580]
**Font Project Git Repo URL:**

**Simple, boring serif:**

**Requirements:**

Hi Fonts people,

I'm an old designer and typographer, not a programmer or software expert, nor a native speaker of English. I have worked with computers since the 80's but I have little to no clue what all these requirements entail so I'm just going to send you the font family I made the last couple of weeks and see if it sticks :)
I'm sure you'll check if it ticks all the boxes anyway. As far as I know I think it's eligible for Google Fonts, but if it isn't, please tell me what to change, and I will. I've been making open source fonts for over twenty years (usually with a Creative Commons Attribution license) and I'd love to have this one added to your library.

cheers, Tup Wanders 
 

Google Fonts will publish only fonts that match its requirements. Please familiarize yourself with the complete documentation in the [Google Fonts Guide](https://googlefonts.github.io/gf-guide/) (GF-Guide) and ensure your font project complies with them before submitting the font family. You can also use the [Google Fonts Project Template](https://github.com/googlefonts/googlefonts-project-template), which will help you create a repository that follows the needed structure and includes build requirements.

By filling this issue, you can confirm the project meets the  requirements (by ticking the cases or putting x between the square brackets in text mode):

- [ ] I read, agree, and comply with the full [contributing requirements](https://googlefonts.github.io/gf-guide/index#pre-production-getting-your-fonts-ready-for-gf)
- [ ] The entire font project is licensed under the [OFL](https://scripts.sil.org/cms/scripts/page.php?site_id=nrsi&id=OFL)
- [ ] There is no "Reserved Font Name" mention in the OFL license
- [ ] Family name is unique according to [namecheck.fontdata.com](https://namecheck.fontdata.com/)
- [ ] The font supports at least [GF Latin Core glyphset](https://github.com/googlefonts/glyphsets/tree/main/GF_glyphsets/Latin/)
- [ ] The repository respect the [upstream repo structure](https://googlefonts.github.io/gf-guide/upstream.html)
- [ ] The source files are available in the repository
- [ ] I am the author of the font, or the author agreed to this submission
- [ ] I will maintain the repository and participate in the onboarding process (addressing, solving, and responding to issues, merging pull requests, etc)

Attach here a pic or a screenshot of the font. One image is enough, it can be a few letters, to give a quick overview.
[Quaaykop.zip](https://github.com/google/fonts/files/12218098/Quaaykop.zip)
![mecanormaQuaaykop](https://github.com/google/fonts/assets/107745782/bb46b648-2735-4261-b040-6c6065d198ed)

**Comment by RosaWagner** [comment]
Hi @ComteDuChagrin thanks for your submission. If the font is accepted we can help with all the set up, no worries. Although, to be able to review it we would need some sources, can you also attach the files from the software editor? (.glyphs, .ufo, .vjf…)

**Comment by ComteDuChagrin** [comment]
Hi Rosa, Do you mean the original files from FontCreator, or the original design I made in Illustrator? Or will either one suffice?

**Comment by ComteDuChagrin** [comment]
No idea what that label means either :)

**Comment by eliheuer** [comment]
Yes, Rosa means the original files from a font editor. Ideally you would use the [Google Fonts Project Template](https://github.com/googlefonts/googlefonts-project-template) to host the source files in a Git repository. 

Usually font sources in the `UFO` and `Glyphs` format are easier to work with, that is why this issue was given the `-- fontforge/fontlab/fontcreator` label.
